jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/service/SummaryService.java

```diff
@@ -0,0 +1,474 @@
+      } else {
+        logger.error("FAILURE [%s]", summaryInstance.getFailureCause());
+      }
+      logger.info(DASHED_LINE);
+    }
+  }
+
+  private void printActions(Summary summaryInstance) {
+    if (StringUtils.isNotBlank(summaryInstance.getActionType()) &&
+        summaryInstance.getActionsRun() != null &&
+        !summaryInstance.getActionsRun().isEmpty()) {
+      logger.info("%s executed : [ %s ]", summaryInstance.getActionType(), String.join(", ", summaryInstance.getActionsRun()));
+    }
+  }
+
+  private void printCommonSummary(Summary summaryInstance, File baseDirectory) {
+    if (summaryInstance.getImageSummariesMap() != null && !summaryInstance.getImageSummariesMap().isEmpty()) {
+      logger.info("Container images:");
+      for (Map.Entry<String, ImageSummary> imageSummaryEntry : summaryInstance.getImageSummariesMap().entrySet()) {
+        printImageSummary(baseDirectory, imageSummaryEntry.getKey(), imageSummaryEntry.getValue());
+      }
+      logger.info("");
+    }
+  }
+
+  private void printImageSummary(File baseDirectory, String imageName, ImageSummary imageSummary) {
+    logger.info(LIST_ELEMENT, imageName);
+    if (imageSummary.getBaseImageName() != null) {
+      logger.info("    * Base image: %s", imageSummary.getBaseImageName());
+    }
+    if (imageSummary.getDockerfilePath() != null) {
+      logger.info("    * Dockerfile image: %s", FileUtil.getRelativeFilePath(baseDirectory.getAbsolutePath(), imageSummary.getDockerfilePath()));
+    }
+    if (imageSummary.getImageStreamUsed() != null) {
+      logger.info("    * ImageStream: %s", imageSummary.getImageStreamUsed());
+    }
+    if (imageSummary.getImageSha() != null) {
+      logger.info("    * SHA: %s", imageSummary.getImageSha());
+    }
+  }
+
+  private void printBanner() {
+    logger.info(" __ / / //_/ / / / _ )/ __/");
+    logger.info("/ // / ,< / /_/ / _  / _/  ");
+    logger.info("\\___/_/|_|\\____/____/___/  \n");
+    logger.info(DASHED_LINE);
+    logger.info("      SUMMARY");
+    logger.info(DASHED_LINE);
+  }
+
+  private void printBuildSummary(Summary summary) {
+    if (summary.getBuildStrategy() != null) {
+      logger.info("Build Strategy : %s", summary.getBuildStrategy());
+    }
+    if (summary.getGeneratorsApplied() != null && !summary.getGeneratorsApplied().isEmpty()) {
+      logger.info("Generators applied: [%s]",  String.join(",", summary.getGeneratorsApplied()));
+    }
+    if (summary.getOpenShiftBuildConfigName() != null) {
+      logger.info("Build config: %s", summary.getOpenShiftBuildConfigName());
+    }
+    logger.info("");
+  }
+
+  private void printResourceSummary(Summary summary, File baseDir) {
+    if (summary.getGeneratedResourceFiles() != null && !summary.getGeneratedResourceFiles().isEmpty()) {
+      if (summary.getEnrichersApplied() != null && summary.getEnrichersApplied().size() < 20) {
+        logger.info("Enrichers applied: [%s]", String.join(",", summary.getEnrichersApplied()));
```

Comment:
Minor:
At the moment I have 
```
[INFO] k8s: Enrichers applied: [jkube-dependency,jkube-controller,jkube-controller-from-configuration,jkube-service,jkube-image,jkube-portname,jkube-project-label,jkube-git,jkube-serviceaccount,jkube-prometheus,jkube-revision-history,jkube-name,jkube-metadata,jkube-pod-annotations,jkube-container-env-java-options]
```
Maybe it would be better with spaces
```
[INFO] k8s: Enrichers applied: [ jkube-dependency, jkube-controller, jkube-controller-from-configuration, jkube-service, jkube-image, jkube-portname, jkube-project-label, jkube-git, jkube-serviceaccount, jkube-prometheus, jkube-revision-history, jkube-name, jkube-metadata, jkube-pod-annotations, jkube-container-env-java-options ]
```